### PR TITLE
Add coverage

### PR DESCRIPTION
**Pull request by PaulineMauryL**

### CODE REVIEW

**Comment by PaulineMauryL** [comment]
## Coverage report


 

_This PR does not seem to contain any modification to coverable code._

**Comment by PaulineMauryL** [comment]
Coverage report: https://htmlpreview.github.io/?https://raw.githubusercontent.com/dscc-admin-ch/lomas/python-coverage-comment-action-data/htmlcov/index.html

(Automatic) Coverage branch: https://github.com/dscc-admin-ch/lomas/tree/python-coverage-comment-action-data

File: server/run_tests_and_coverage.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+# On admin_database: "mongodb" and dataset_store: "LRU_cache" and production mode
+export LOMAS_TEST_MONGO_INTEGRATION=1
+coverage run --source=. -m unittest discover -s .
+ret=$?
+
+# On admin_database: "yaml" and dataset_store: "basic" and developer mode
+export LOMAS_TEST_MONGO_INTEGRATION=0
+coverage run -a --source=. -m unittest discover -s .
+ret=$?
```
Comment (damienbfs) [inline server/run_tests_and_coverage.sh:17]: You are overwritting the results of the first tests here. You might want "and" both.

Comment (PaulineMauryL) [reply server/run_tests_and_coverage.sh:17]: you mean for the coverage ? the -a should append to the coverage file.

do you mean for ret=$?

Comment (PaulineMauryL) [reply server/run_tests_and_coverage.sh:17]: ah yes for ret, yes i should try to 'and' it indeed

Comment (PaulineMauryL) [reply server/run_tests_and_coverage.sh:17]: if it is not possible, i can also let the tests separately in github actions as before and do another one just to check the coverage

Comment (PaulineMauryL) [reply server/run_tests_and_coverage.sh]: nice catch ^^

**Review by damienbfs — APPROVED** [review]

**Pull request merged**